k8s/runners/staging/large-arm-pub/release.yaml

```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: runner-arm-staging-patch
+  namespace: gitlab
+  annotations:
+    cd.spack.io/staged-resource: "1"
+data:
+  apiVersion: helm.fluxcd.io/v1
+  kind: HelmRelease
+  name: runner-large-arm-pub
+  patch: |
+    - op: replace
+      path: /metadata/name
+      value: runner-large-arm-pub-{ENV}
+    - op: replace
+      path: /spec/chart/version
+      value: "0.37.2"  # bleeding edge
+    - op: replace
+      path: /spec/releaseName
+      value: runner-large-arm-pub-{ENV}
+    - op: replace
+      path: /spec/values/replicas
+      value: 1
+    - op: replace
+      path: /spec/values/gitlabUrl
+      value: "https://gitlab.{ENV}.spack.io/"
+    - op: replace
+      path: /spec/values/terminationGracePeriodSeconds
+      value: 300  # five minutes
+    - op: replace
+      path: /spec/values/concurrent
+      value: 3
+    - op: replace
+      path: /spec/values/checkInterval
+      value: 1
+    - op: replace
+      path: /spec/values/runners/tags
+      value: "arm,aarch64,graviton,graviton2,small,medium,large,huge,{ENV},public,aws,spack"
```

Comment:
FYI, to hopefully make this easier to review -
This file is identical to the staging patch for the x86 runner `release.yaml` with two differences -
1) All references to `runner-x86-*`, etc are replaced by `runner-arm-*` (obviously)
2) This line highlighted by this comment, i.e. the patch for `values.runners.tags`, was changed to the equivalent value from `k8s/runners/large-arm-pub/release.yaml`, but with `{ENV}` added in.